app/views/vms/index.html.erb

```diff
@@ -58,6 +59,9 @@
       <td class="table-active align-middle">
         <%= render 'vm_status', vm: vm %>
       </td>
+      <td class="d-none"> <!-- Also not intended to be shown! -->
+        <%= vm.ensure_config.description %>
```

Comment:
Dont use ensure_config here
This is not the place to create config objects, that would be the controller.
use config instead and do a nil check with
@vm.config&.description
Otherwise we would create unneccessary config objects

Reply:
Thanks, changed it.